admin/src/components/AbuseReportsPopUp/index.js

```diff
@@ -0,0 +1,145 @@
+        label={formatMessage({ id: `${pluginId}.popup.reports.item.button` })}
+      /></td>
+    </tr>);
+}
+
+const AbuseReportsPopUp = ({ isOpen, reports, comment, blocked, blockedThread, onBlockClick, onBlockThreadClick, onClose, onAbuseReportResolveClick }) => {
+
+  const { formatMessage } = useGlobalContext();
+
+  const resolveReportReason = reason => formatMessage({ id: `${pluginId}.popup.reports.item.reason.${reason.toLowerCase()}`});
+
+  const headers = [
+    {
+      name: formatMessage({ id: `${pluginId}.popup.reports.item.header.date` }),
+      value: 'created_at',
+    },
+    {
+      name: formatMessage({ id: `${pluginId}.popup.reports.item.header.reason` }),
+      value: 'reason',
+    },
+    {
+      name: formatMessage({ id: `${pluginId}.popup.reports.item.header.message` }),
+      value: 'content',
+    },
+    { value: 'actions' }
+  ];
+
+  const rows = reports.map(({ id, created_at, content, reason }) => ({
+    id,
+    created_at: moment(created_at).format('DD/MM/YYYY, HH:mm'),
+    reason: resolveReportReason(reason),
+    content,
+  }));
+  
+  return (
+    <Modal isOpen={isOpen} onToggle={onClose}>
+      <HeaderModal>
+        <section>
+          <HeaderModalTitle>
+            <FormattedMessage id={`${pluginId}.popup.reports.header`} />
+          </HeaderModalTitle>
+        </section>
+      </HeaderModal>
+      <ModalForm>
+        <ModalBody>
+          <section style={{ width: '100%' }}>
+            <ItemPreviewContainer>
+              <ItemDetails root={true} {...comment} />
+            </ItemPreviewContainer>
+            { isEmpty(reports) && (
+              <EmptyView>
+                <FontAwesomeIcon icon={faSmile} size="5x" />
+                <FormattedMessage id={`${pluginId}.popup.reports.empty`} />
+              </EmptyView>
+            ) }
+            { !isEmpty(reports) && (<TableContainer>
+                <Table
+                customRow={props => (<CustomRow {...props} formatMessage={formatMessage} onClick={(e) => onAbuseReportResolveClick(e, props.row.id)} />)}
+                headers={headers}
+                rows={rows}
+              />
+            </TableContainer>)}
+          </section>
+        </ModalBody>
+      </ModalForm>
+      { !isEmpty(reports) && !blockedThread && !blocked && (<ModalFooter>
+        <section>
+          <label>
+            <FormattedMessage id={`${pluginId}.popup.reports.footer.actions`} />
+          </label>
+          { !blockedThread && (<ButtonModal
+            onClick={e => onBlockClick(comment.id)}
+            isSecondary
+            message={`${pluginId}.list.item.moderation.button.comment.hide`} />)}
+          <ButtonModal 
+            onClick={e => onBlockThreadClick(comment.id)}
+            isSecondary
+            message={`${pluginId}.list.item.moderation.button.thread.hide`} />
+        </section>
+      </ModalFooter>)}
+    </Modal>
+  );
+}
+
+AbuseReportsPopUp.propTypes = {
```

Comment:
propTypes are not aligned with actual props used, copy paste from other place?

I think there should be an eslint plugin for detecting this automatically